backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/ProprietaryService.java

```diff
@@ -33,7 +33,7 @@ public Proprietary loadProprietaryFromNorm(LoadProprietaryFromNormUseCase.Query
   }
 
   @Override
-  public Proprietary updateProprietaryFromNorm(UpdateProprietaryFromNormUseCase.Query query)
+  public Proprietary updateProprietaryFrameFromNorm(UpdateProprietaryFromNormUseCase.Query query)
```

Comment:
Could you remove this throwing of the NormNotFoundException. I wrongly added last week I think. Probably you also need to remove it from the interface. The exception is an unchecked exception